### DISCUSSION TITLE
Move functions out of schema repo

### DISCUSSION THREAD

**Discussion by saskiad** [AllenNeuralDynamics/aind-data-schema#689] [Ideas]
There are a lot of functions to do things like update data descriptions and stuff like that that I'm not sure belong in this repo. Or perhaps should be organized differently than they currently are?

**Comment by jtyoung84** [comment]
I like the idea of putting the schema upgrade code into its package "aind-data-schema-upgrader" or something like that

**Reply by dyf** [reply]
I'm good with this too. I would call it `aind-metadata-upgrader`.
